### PR TITLE
Confess improvements

### PR DESCRIPTION
**Pull request by obiwac**

### CODE REVIEW

File: src/commands/confess.js [hunk 1]
```diff
@@ -54,7 +65,7 @@ module.exports = {
             if (reaction.emoji.name === IN_FAVOUR_REACTION) {
                 await vote.edit({
```
Comment (NoaQuenon) [inline src/commands/confess.js:69]: The editing should be done after sending the confession in the confess channel (the two blocks should be inverted)

**Review by NoaQuenon — CHANGES_REQUESTED** [review]

File: src/commands/confess.js (hunk 1 above)
Comment (Emilevillette) [reply src/commands/confess.js:69]: should be resolved in [2590894](https://github.com/obiwac/DiscordSINFBot/pull/3/commits/25908945e152444aa240b94f98d84c1d065686fd) , can you review and confirm ?

Comment (NoaQuenon) [reply src/commands/confess.js:69]: That should be fine

**Review by NoaQuenon — APPROVED** [review]

**Review by Emilevillette — APPROVED** [review]
See previous comments.

**Pull request merged**